CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.15.0 (2020-06-15)
+
+- Updated the lookup component to include a button which triggers the show more picker from the input box. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)
```

Comment:
```suggestion
- Updated the lookup component to include a search button in the input field that opens the show more picker. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)
```
"search button" and "input field" are the terminology in the design guidelines. If we actually mean the